types/index.d.ts

```diff
@@ -25,10 +25,15 @@ interface StatusBar {
     overlaysWebView(isOverlay: boolean): void;
 
     /**
-    * Use the default statusbar (dark text, for light backgrounds).
+    * Automatically chooses light or dark content based on the user interface style
```

Comment:
This doesn't match the other descriptions where it says the automatic style is for iOS only.

But I think in Android it should also behave the same way as iOS, changing the status bar style depending on the device style

Reply:
I did not find in the documentation that it has the same behaviour as iOS, and in the source code the method to change the status bar content style is `setAppearanceLightStatusBars(boolean isLight)`
So I think that in Android we won't have a content style default/automatic.

Reply:
yeah, there is no specific value that does it automatically for you, but could check the actual device theme on app start if set to default to set the status bar style based on the device theme and also check `onConfigurationChanged` method for detecting changes on the configuration to re set the status bar style according to the current device theme